<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STWBInviteFriendClass</class>
 <widget class="QDialog" name="STWBInviteFriendClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>333</width>
    <height>466</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STWBInviteFriend</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widTitle" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>25</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget#widTitle{background-color:#484950;border:0px;}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>邀请好友</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>785</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="minimumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
QPushButton{border-image: url(:/SoftTerminal/images/close.png);}

QPushButton:hover:!pressed {
	
	background-color: #ff6741;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="styleSheet">
      <string notr="true">
QScrollBar::vertical
{
    width:4px;
    background:rgb(0,0,0,0%);
    margin:0px,0px,0px,0px;
    padding-top:0px;   /*上预留位置*/
    padding-bottom:0px;    /*下预留位置*/
}

/*滚动条中滑块的样式*/
QScrollBar::handle::vertical
{
    width:4px;
    background:rgb(0,0,0,25%);
    border-radius:2px;
    min-height:20px;
}

/*鼠标触及滑块样式*/
QScrollBar::handle::vertical::hover
{
    width:5px;
    background:rgb(0,0,0,50%);
    border-radius:2px;
    min-height:20;
}

/*设置下箭头*/
QScrollBar::add-line:vertical
{
    height:0px;
    subcontrol-position:bottom;
}

/*设置上箭头*/
QScrollBar::sub-line:vertical
{
    height:0px;
    subcontrol-position:top;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
    background: #3e414a;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="lwFriendList">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QListWidget{
    background-color: #3e414a;   
    font:10pt &quot;微软雅黑&quot;;  
	color: #ffffff;
    border:0px;
	border-left:1px solid #484950;
	border-right:1px solid #484950;
}
QListWidget::Item{
	height:45px;padding-top:0px; padding-bottom:0px;
	border-bottom: 1px solid #494c54;
}</string>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::NoSelection</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widBottom" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget#widBottom{background-color:#484950;}</string>
     </property>
     <layout class="QHBoxLayout" name="hblayout">
      <property name="spacing">
       <number>50</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pbOK">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:pressed {
	background-color: rgb(48, 186, 120);
}
QPushButton:hover:!pressed {
	
	background-color: rgb(109, 206, 160);
}</string>
        </property>
        <property name="text">
         <string>是</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbCancel">
        <property name="minimumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>65</width>
          <height>25</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">

QPushButton {
	border-radius: 3px;
	color: rgb(255, 255, 255);
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:pressed {
	background-color: rgb(255, 103, 65);
}
QPushButton:hover:!pressed {
	background-color: rgb(255, 133, 103);
}</string>
        </property>
        <property name="text">
         <string>否</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/softterminal.qrc"/>
 </resources>
 <connections/>
</ui>
